docs-src/content/audio-input/_index.md

```diff
@@ -138,7 +154,15 @@ while haveAudioData:
 {{< /tab >}}
 
 {{< tab "C++" "c++" >}}
-// Example coming soon!
+while(haveAudioData) {
```

Comment:
Should there be a space between `while` and `(`?

Reply:
Yes, there probably should be if we are following the style guide :)